scripts/training/prepare-config.py

```diff
@@ -86,3 +86,91 @@
     yaml.dump(config, yaml_file, default_flow_style=False, default_style='\'', explicit_start=True)
 
 s3_client.upload_file(Bucket=s3_bucket, Key=yaml_key, Filename=local_yaml_path)
+
+
+# Training with different configurations on each worker (aka Multi Config training)
+config['MULTI_CONFIG'] = os.environ.get('DR_MULTI_CONFIG', 'False')
```

Comment:
Let's call the variable DR_TRAIN_MULTI_CONFIG as it only applies to training.

Reply:
Agreed, done